flights/berks_ceh_insurances_flight.yaml

```diff
@@ -114,8 +114,8 @@ entityDefinitions:
         - !<transforms.ConcatTransform>
           columns: ["SCHIP", "NoSCHIPReason"]
           separator: "_"
-      general.status:
-        type: "general.status"
+      ol.statusdetails:
```

Comment:
same comment. Please do for all insurance entity sets.

Reply:
in new 317ce78